static/js/app.js

```diff
@@ -800,15 +763,11 @@ $(function () {
             }
         },
         getCurrentUrlParams: function () {
-          var dateRange = app.model.get("dateRange");
-          var center = app.map.getCenter();
-          return "start_date=" + moment(dateRange[0]).format("YYYY-MM-DD") +
-              "&end_date=" + moment(dateRange[1]).format("YYYY-MM-DD") +
-              "&show_fatal=" + (parseInt(app.model.get("showFatal")) || 0) +
-              "&show_severe=" + (parseInt(app.model.get("showSevere")) || 0) +
-              "&show_light=" + (parseInt(app.model.get("showLight")) || 0) +
-              "&show_inaccurate=" + (parseInt(app.model.get("showInaccurateMarkers")) || 0) +
-              "&zoom=" + app.map.zoom + "&lat=" + center.lat() + "&lon=" + center.lng();
+            var dateRange = app.model.get("dateRange");
+            var center = app.map.getCenter();
+            return "start_date=" + moment(this.dateRanges[0]).format("YYYY-MM-DD") +
+                "&end_date=" + moment(this.dateRanges[1]).format("YYYY-MM-DD") +
+                "&zoom=" + app.map.zoom + "&lat=" + center.lat() + "&lon=" + center.lng();
```

Comment:
You're right. But instead of having some 20 additional URL parameters I'd like to implement a more elegant way.
Opened an issue yesterday - #423